### DISCUSSION TITLE
TypeError: undefined is not an object (evaluating 'NativeUnimoduleProxy.viewManagersNames.includes')

### DISCUSSION THREAD

**Discussion by Akumzy** [expo/expo-cli#4361] [Q&A]
After upgrading to SDK 45 I started getting this error and it's driving me nut

```log
Invariant Violation: "main" has not been registered. This can happen if:
* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
* A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called.

```
<img width="974" alt="Screenshot 2022-05-10 at 2 18 30 PM" src="https://user-images.githubusercontent.com/25849232/167658010-53f0ab3f-82fe-458b-b0c3-1c09055ac5f6.png">

**Comment by KaabAn** [comment]
Hello,
did you resolve this issue pls ? and how ?

Thank you

**Reply by Akumzy** [reply]
Hi,
Sorry I can't remember how I solved it.
